NEP-P2P_Protocol_change.mediawiki

```diff
@@ -0,0 +1,129 @@
+<pre>
+  NEP: <to be assigned>
+  Title: P2P Protocol change
+  Author: Shargon and Belane
+  Type: Standard
+  Status: Draft
+  Created: 2018-03-07
+</pre>
+
+==Abstract==
+
+We believe that a P2P protocol optimization is necessary in order to avoid unnecessary data exchange between nodes.
+In this sense, reducing the computational consumption and, consequently, improving and speeding up the communication between nodes.
+
```

Comment:
```suggestion
This proposal describes various improvements.
```
In general an abstract should be standalone readable and trigger the user to want to read more/go in depth. I believe this small addition gives that last bit of information that can trigger.